app/domain/logs.rb

```diff
@@ -29,50 +29,58 @@ module Security
 
       end
 
-      module AuthnOidc
-
-        ExtractedUsernameFromIDToked = ::Util::TrackableLogMessageClass.new(
-          msg: "Extracted username '{0}' from ID Token field '{1-id-token-username-field}'",
-          code: "CONJ00004D"
-        )
+      module OAuth
 
-        TokenDecodeSuccess = ::Util::TrackableLogMessageClass.new(
-          msg: "Token decode succeeded",
-          code: "CONJ00005D"
-        )
-
-        TokenDecodeFailed = ::Util::TrackableLogMessageClass.new(
-          msg: "Failed to decode the token with the error '{0-exception}'",
-          code: "CONJ00018D"
-        )
-
-        OIDCProviderUri = ::Util::TrackableLogMessageClass.new(
+        IdentityProviderUri = ::Util::TrackableLogMessageClass.new(
           msg: "Working with Identity Provider {0-provider-uri}",
           code: "CONJ00007D"
         )
 
-        OIDCProviderDiscoverySuccess = ::Util::TrackableLogMessageClass.new(
+        IdentityProviderDiscoverySuccess = ::Util::TrackableLogMessageClass.new(
           msg: "Identity Provider discovery succeeded",
           code: "CONJ00008D"
         )
 
+        IdentityProviderCertificateFetchedFromCache = ::Util::TrackableLogMessageClass.new(
+          msg: "Identity Provider certificates fetched successfully from cache",
+          code: "CONJ00017D"
+        )
+
         FetchProviderCertsSuccess = ::Util::TrackableLogMessageClass.new(
           msg: "Fetched Identity Provider certificates successfully",
           code: "CONJ00009D"
         )
 
-        OIDCProviderCertificateFetchedFromCache = ::Util::TrackableLogMessageClass.new(
-          msg: "Identity Provider certificates fetched successfully from cache",
-          code: "CONJ00017D"
-        )
-
         ValidateProviderCertificateIsUpdated = ::Util::TrackableLogMessageClass.new(
           msg: "Validating Identity Provider certificates are up to date",
           code: "CONJ00019D"
         )
 
       end
 
+      module Jwt
+
+        TokenDecodeSuccess = ::Util::TrackableLogMessageClass.new(
+          msg: "Token decode succeeded",
+          code: "CONJ00005D"
+        )
+
+        TokenDecodeFailed = ::Util::TrackableLogMessageClass.new(
+          msg: "Failed to decode the token with the error '{0-exception}'",
+          code: "CONJ00018D"
+        )
+
+      end
+
+      module AuthnOidc
+
+        ExtractedUsernameFromIDToked = ::Util::TrackableLogMessageClass.new(
+          msg: "Extracted username '{0}' from ID Token field '{1-id-token-username-field}'",
```

Comment:
Let's have Shuli go through all these log/error messages so that they are uniform

Reply:
I'll pass them by shuli later on in one PR so it will be easier.